libs/output-mapping/src/Writer/Table/TableDefinition/TableDefinitionColumnFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Writer\Table\TableDefinition;
+
+use Keboola\Datatype\Definition\Common;
+
+class TableDefinitionColumnFactory
```

Comment:
a vlastně ani tahle :) ale jestli to @ErikZigo přijde v pohodě, tak 🤷

Reply:
jj, to stejny.  muzes vidit proc je to takhle na ten posledni PR

Reply:
Nechme tak